test/fixtures/v5/arrive/no_modifier.json

```diff
@@ -6,6 +6,7 @@
         "name": "Street Name"
     },
     "instructions": {
+        "ar": "لقد وصلت إلى الوجهة ",
```

Comment:
Is the trailing whitespace here expected?

Reply:
Here’s the format string:

https://github.com/Project-OSRM/osrm-text-instructions/blob/791335b25f7a6d686c522294ca8a06c7da3064fa/languages/translations/ar.json#L62

`{nth}` resolves to the empty string, but it appears at the end of the format string, so it doesn’t get collapsed down:

https://github.com/Project-OSRM/osrm-text-instructions/blob/b692dfb39b521e91ec6d0207065c91072c3b31a4/index.js#L286

The same issue affects Vietnamese. I guess technically OSRMTI could trim the final message if it causes problems for clients.